eng/pipelines/templates/jobs/ci.tests.yml

```diff
@@ -68,3 +71,10 @@ jobs:
       arguments: >
         -PackageInfoDirectory '$(Build.ArtifactStagingDirectory)/PackageInfo'
         -Toolchain '$(Toolchain)'
+
+  - ${{ if eq(parameters.TestProxy, true) }}:
```

Comment:
Could we only do this if `System.Debug` is set too? That's a lot of output. At the very least, collapse it into a region like you do for other tools.

Reply:
I'm not sure that collapsing it will do much as it's already isolated to its own task.  I like the idea of conditioning it on System.Debug